packages/webpack-extraction-plugin/__fixtures__/webpack/mixed/output.ts

```diff
@@ -1,9 +1,11 @@
-import { __resetStyles, __styles, __resetCSS, __css } from '@griffel/react';
-export const useClasses = __css({
+import { __css as _css } from '@griffel/react';
+import { __resetCSS as _resetCSS } from '@griffel/react';
+import { __resetStyles, __styles } from '@griffel/react';
+export const useClasses = _css({
   root: {
     sj55zd: 'fe3e8s9',
   },
 });
-export const useClassName = __resetCSS('rjefjbm', 'r7z97ji');
+export const useClassName = _resetCSS('rjefjbm', 'r7z97ji');
 
-require('./code.griffel.css!=!../../../virtual-loader/index.js?style=%2F**%20%40griffel%3Acss-start%20%5Bd%5D%20**%2F%0A.fe3e8s9%7Bcolor%3Ared%3B%7D%0A%2F**%20%40griffel%3Acss-end%20**%2F%0A%2F**%20%40griffel%3Acss-start%20%5Br%5D%20**%2F%0A.rjefjbm%7Bcolor%3Ared%3Bpadding-left%3A4px%3B%7D.r7z97ji%7Bcolor%3Ared%3Bpadding-right%3A4px%3B%7D%0A%2F**%20%40griffel%3Acss-end%20**%2F%0A!./code.ts');
+require('./code.griffel.css!=!../../../virtual-loader/index.js?style=%2F**%20%40griffel%3Acss-start%20%5Br%5D%20**%2F%0A.rjefjbm%7Bcolor%3Ared%3Bpadding-left%3A4px%3B%7D.r7z97ji%7Bcolor%3Ared%3Bpadding-right%3A4px%3B%7D%0A%2F**%20%40griffel%3Acss-end%20**%2F%0A%2F**%20%40griffel%3Acss-start%20%5Bd%5D%20**%2F%0A.fe3e8s9%7Bcolor%3Ared%3B%7D%0A%2F**%20%40griffel%3Acss-end%20**%2F%0A!./code.ts');
```

Comment:
CSS in the query is same, only order was changed. It does matter as Webpack plugin will sort it later.